composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "Offers the Bread financing and checkout tools for your Magento store",
     "license": "MIT",
     "minimum-stability": "stable",
-    "version": "1.1.7",
+    "version": "1.1.7.2",
```

Comment:
Doubt this will pass magento check, as far as I know it can three numbers only MAJOR.MINOR.PATCH
https://devdocs.magento.com/guides/v2.3/extension-dev-guide/versioning/

Reply:
yeah this was only added in so I could tell this apart from our actual plugin versions on the merchant's staging sites, this'll be removed before it's an actual PR 👍